packages/hop-node/package.json

```diff
@@ -45,9 +45,11 @@
     "@eth-optimism/message-relayer": "^0.2.4",
     "@eth-optimism/watcher": "0.0.1-alpha.9",
     "@ethersproject/hdnode": "^5.0.9",
+    "@fxportal/maticjs-fxportal": "^1.1.1",
     "@hop-protocol/core": "0.0.1-beta.57",
     "@hop-protocol/sdk": "0.0.1-beta.286",
-    "@maticnetwork/maticjs": "2.0.45",
+    "@maticnetwork/maticjs": "^3.2.0",
```

Comment:
My loose opinion would be to pin all the Matic packages, as we've seen things break unintentionally with upgrades.

Reply:
Good idea. 

Resolved with https://github.com/hop-protocol/hop/pull/384/commits/fef0c0adff680b369340f9b219ca3b5acbd002b8